key.core/src/main/java/de/uka/ilkd/key/api/Matcher.java

```diff
@@ -67,7 +68,7 @@ public List<VariableAssignments> matchPattern(String pattern, Sequent currentSeq
         Taclet t = parseTaclet(patternString, copyServices);
 
         // Build Matcher for Matchpattern
-        LegacyTacletMatcher ltm = new LegacyTacletMatcher(t);
+        VMTacletMatcher ltm = new VMTacletMatcher(t);
```

Comment:
```suggestion
        VMTacletMatcher tm = new VMTacletMatcher(t);
```
The variable should probably be renamed